README.md

```diff
@@ -14,21 +51,25 @@ Parse in Node.js
 ```javascript
 const parser = require('screwdriver-config-parser');
 
-parser({
-    // Configuration (in YAML form)
-    yaml: fs.readFileSync('sample.yaml', 'utf-8'),
-    // Name of current job
-    jobName: 'main'
-}, (err, data) => {
-    // data.execute contains the commands to execute
-    fs.writeFileSync('execute.json', JSON.stringify(data.execute));
+// Configuration (in YAML form)
+parser(fs.readFileSync('sample.yaml', 'utf-8'), (err, pipeline) => {
+    // Workflow for the pipeline
+    // pipeline.workflow
+
+    // All the main jobs with the steps to execute and environment variables to set
+    // pipeline.jobs.main[].steps
+    // pipeline.jobs.main[].environment
+
+    // All the publish jobs with the steps to execute and environment variables to set
+    // pipeline.jobs.publish[].steps
+    // pipeline.jobs.publish[].environment
 });
 ```
 
 Or via the command line
 
 ```bash
-config-parse path/to/screwdriver.yaml job-name path-to-artifacts-dir
+config-parse path/to/screwdriver.yaml job-name environment-permutation
```

Comment:
Can we be more clear on this? What should the environment-permutation be? Maybe give  a specific example?